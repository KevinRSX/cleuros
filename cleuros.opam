# This file is generated by dune, edit dune-project instead
opam-version: "2.0"
synopsis: "plt project"
description: "plt project"
maintainer: ["Samuel Meshoyrer"]
authors: ["Samuel Meshoyrer"]
license: "LICENSE"
tags: ["topics" "to describe" "your" "project"]
homepage: "https://github.com/smeshoyrer/cleuros"
doc: "https://github.com/smeshoyrer/cleuros"
bug-reports: "https://github.com/smeshoyrer/cleuros/issues"
depends: [
  "ocaml"
  "dune" {>= "3.0"}
  "odoc" {with-doc}
]
build: [
  ["dune" "subst"] {dev}
  [
    "dune"
    "build"
    "-p"
    name
    "-j"
    jobs
    "@install"
    "@runtest" {with-test}
    "@doc" {with-doc}
  ]
]
dev-repo: "git+https://github.com/smeshoyrer/cleuros.git"
