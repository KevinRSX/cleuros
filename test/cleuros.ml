open Cleuros_parser

let () = 
  print_int (Cleuros.eval (Var "x")); print_newline (); 
  print_int (Cleuros.eval (Int 54)); print_newline ();
  print_int (Cleuros.eval (EOF)); print_newline();