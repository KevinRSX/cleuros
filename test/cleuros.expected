456
54
123
